specs/core/0_beacon-chain.md

```diff
@@ -1013,6 +1013,9 @@ def on_startup(initial_validator_entries: List[Any],
 
     # Setup state
     initial_shuffling = get_new_shuffling(ZERO_HASH, initial_validator_registry, 0)
+    validator_indices = list(range(len(initial_validator_registry)))
```

Comment:
I'm not sure this quite works because the `initial_validator_registry` can have validators pending activation (because their balance is <32 ETH).

Reply:
Ah, your right. Need to cherry pick the active indices

Reply:
I think `get_active_validator_indices(initial_validator_registry)` is what you're looking for?